### PR TITLE
Redesign of the main landing page.

### PR DESCRIPTION
**Pull request by hhhhhhhhhn**
It seemed like it needed modernization, so a more modern design was applied to the same content, with an animated background (not shown in small devices), with proper mobile device support and a less old and static feel. (Quick fix before [this ](https://github.com/SFTtech/openage-pr/pull/7) pull)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,105 +5,125 @@
         <meta http-equiv="X-UA-Compatible" content="chrome=1">
         <meta name="viewport" content="width=640" />
 
-        <link rel="stylesheet" href="web/stylesheets/core.css" media="screen"/>
-        <link rel="stylesheet" href="web/stylesheets/mobile.css" media="handheld, only screen and (max-device-width:640px)"/>
-        <link rel="stylesheet" href="web/stylesheets/pygment_trac.css"/>
+        <link rel="stylesheet" href="web/stylesheets/main.css"/>
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans:300&display=swap" rel="stylesheet">
```
Comment (simonsan) [inline index.html]: Can you please download that and move that to `web/fonts/`. So we don't expose the visitors to Google tracking.

File: index.html [hunk 2]
```diff
@@ -5,105 +5,125 @@
...
+            var h = window.innerHeight;
+            if (w/h < 1.45){         //if aspect ratio is small enough
+                styleSheet.type = "text/css";
+                styleSheet.innerText = styles;
+                document.head.appendChild(styleSheet);
+            }
+
+        
+        //////////////////////Javscript end/////////////////////////////
+        </script>
```
Comment (simonsan) [inline index.html]: Can you please move the `script`-part to `web/js/` and include it from there, so we have everything at one place.

**Review by simonsan — CHANGES_REQUESTED** [review]
Thank you, looks nice. 👍 I left you two review comments: One for external dependencies and the other one for embedded css/javascript.

File: index.html (hunk 2 above)
Comment (goto-bus-stop) [reply index.html]: Can this be done with a CSS media query instead? 
The [`aspect-ratio`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/aspect-ratio) query is supported pretty universally.

```html
<style>
@media (max-aspect-ratio: 3/2) {
  /* put the css here */
}
</style>
```

Comment (hhhhhhhhhn) [reply index.html]: I forgot that existed, just finished the fixes

**Review by simonsan — APPROVED** [review]
Thanks! ;-)

**Review by TheJJ — APPROVED** [review]
haha very cool, especially with the video :) thank you very much!

**Pull request merged**